docs/Skill_Tutorial.md

```diff
@@ -184,7 +184,7 @@ How do I feel?
 You are $_mood$.
 ```
 
-In this example, SUSI remembers your mood and can tell you about it. The actual word which was used to describe the mood was never printed to the user before because using the `^^` symbols it got quoted and became invisible.
+In this example, SUSI.AI remembers your mood and can tell you about it. The actual word which was used to describe the mood was never printed to the user before because using the `^^` symbols it got quoted and became invisible.
```

Comment:
@sachin235 shoildn't there be a comma between _using the '^^' symbols_ and _it got quoted_ ?

Reply:
@AdityaSrivast 
Thanks for your feedback. I've incorporated all the suggested changes.